rules/windows/collection_winrar_encryption.toml

```diff
@@ -14,6 +14,40 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Encrypting Files with WinRar or 7z"
+note = """## Triage and analysis
+
+### Investigating Encrypting Files with WinRar or 7z
+
+Attackers may compress and/or encrypt data collected before exfiltration. Compressing the data can help obfuscate the
+collected data and minimize the amount of data sent over the network. Encryption can be used to hide information that is
+being exfiltrated from detection or make exfiltration less apparent upon inspection by a defender.
+
+These steps are usually done in preparation for exfiltration, meaning the attack may be in its final stages.
+
+#### Possible investigation steps:
+
+- Investigate script execution chain (parent process tree).
+- Retrieve the encrypted file.
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Check if the password used in the encryption was included in the command line.
+- Decrypt the rar/zip and check if the information is sensitive.
+- If the password is not available, and the format is ZIP or the option used in WinRAR is not the `-hp`, file names
+included in the encryption can be listed.
```

Comment:
Should this investigation step have an action associated with it?

Reply:
Yeah, I'll rephrase it

Reply:
https://github.com/elastic/detection-rules/pull/1822/commits/5b26480ac850c73df56cfe60e01615469e640f68